src/static/index.js

```diff
@@ -0,0 +1,80 @@
+function handleEditClick(event) {
+    const rowId = getRowId(event.srcElement.id);
+    // Toggle visibilities of buttons.
+    toggleVisible("cancel-button-"+rowId);
+    toggleAllOfSame("button[id^='edit-button-']");
+    toggleAllOfSame("button[id^='submit-button-" + CSS.escape(rowId) + "']");
+    toggleAllOfSame("div[id$=" + CSS.escape(rowId) + "]");
+};
+
+// Get a NodeList Obj of all edit buttons, add EventListener to each.
+var editButtons = document.querySelectorAll("button[id^='edit-button-']");
```

Comment:
Generally speaking, the `var` keyword should be avoided in modern JavaScript because of its weird scoping rules. There's a good explanation here: https://eslint.org/docs/latest/rules/no-var.